demo/index.html

```diff
@@ -9,251 +9,247 @@
 subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
 -->
 <html>
-
 <head>
 
   <title>iron-dropdown</title>
 
-  <script src="../../../@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
-  <script src="../../../web-animations-js/web-animations-next-lite.min.js"></script>
-
-  <script type="module" src="../../iron-image/iron-image.js"></script>
-  <script type="module" src="../../iron-demo-helpers/demo-snippet.js"></script>
-  <script type="module" src="../../iron-demo-helpers/demo-pages-shared-styles.js"></script>
-
-  <script type="module" src="./x-select.js"></script>
-
-  <!-- FIXME(polymer-modulizer):
-        These imperative modules that innerHTML your HTML are
-        a hacky way to be sure that any mixins in included style
-        modules are ready before any elements that reference them are
-        instantiated, otherwise the CSS @apply mixin polyfill won't be
-        able to expand the underlying CSS custom properties.
-        See: https://github.com/Polymer/polymer-modulizer/issues/154
-        -->
-    <script type="module">
-const $_documentContainer = document.createElement('template');
-
-$_documentContainer.innerHTML = `<custom-style><style is="custom-style" include="demo-pages-shared-styles">
-    demo-snippet {
-      --demo-snippet-code: {
-        max-height: 250px;
-      }
-    }
-
-    [slot="dropdown-content"] {
-      background-color: white;
-      line-height: 20px;
-      border-radius: 3px;
-      box-shadow: 0px 2px 6px #ccc;
-    }
-
-    .random-content {
-      padding: 1.5em 2em;
-      max-width: 250px;
-    }
-
-    button {
-      border: 1px solid #ccc;
-      background-color: #eee;
-      padding: 1em;
-      border-radius: 3px;
-      cursor: pointer;
-    }
-
-    button:focus {
-      outline: none;
-      border-color: blue;
-    }
-
-    ul, li {
-      display: block;
-      position: relative;
-      margin: 0;
-      padding: 0;
-    }
-
-    a {
-      display: block;
-      position: relative;
-      padding: 1em;
-      text-decoration: none;
-    }
-
-    li:not(:last-of-type) {
-      border-bottom: 1px solid #eee;
-    }
-
-    a:hover {
-      text-decoration: underline;
-    }
+  <script src="https://unpkg.com/@webcomponents/webcomponentsjs@^2.0.0/webcomponents-loader.js"></script>
+  <script src="https://unpkg.com/web-animations-js@^2.3.1/web-animations-next-lite.min.js"></script>
 
-    iron-image {
-      padding: 1em;
-      background-color: #fff;
-      box-shadow: 0px 2px 6px #ccc;
-      border-radius: 3px;
-    }
-  </style></custom-style>`;
-
-document.body.appendChild($_documentContainer.content);
-</script>
+  <script type="module">
+    import '@polymer/iron-image/iron-image.js';
+    import '@polymer/iron-demo-helpers/demo-snippet.js';
+    import '@polymer/iron-demo-helpers/demo-pages-shared-styles.js';
+    import './x-select.js';
+  </script>
+
+  <!--
+    FIXME(polymer-modulizer):
```

Comment:
undo this modulizer hack (see doc)

Reply:
This one seems to be needed here. Removing it changes the styles for the demo.

Reply:
Did you try `ShadyCSS.styleDocument()`?

Reply:
I'll give it another shot; I just tried the naive way to undo this here.

Reply:
(posted that before seeing your reply) No, I didn't, I'll try that and see what happens.

Reply:
I was able to revert a good portion of these modulizer changes. I needed to keep the one which creates a `<custom-style>` with an `include` attribute as script so that I could import the referenced style before it starts up. (Which feels redundant? :/ )